### PR TITLE
Replace "WFS3" with "OAPIF"

### PR DESCRIPTION
**Pull request by DelazJ**
refs https://github.com/qgis/QGIS-Documentation/pull/6213#issuecomment-1035977980
@elpaso I made an almost blind replacement. Except for images and QGIS_SERVER_API_WFS3_MAX_LIMIT

### CODE REVIEW

File: docs/server_manual/services/ogcapif.rst [hunk 1]
```diff
@@ -438,7 +438,7 @@ A typical Linux installation will have the following directory tree:
         │   ├── jsonFormatter.min.js
         │   └── style.css
         └── templates
-            └── wfs3
+            └── oapif
```
Comment (elpaso) [inline docs/server_manual/services/ogcapif.rst:441]: this should be updated in QGIS sources...

**Review by elpaso — APPROVED** [review]
LGTM except what noted.

File: docs/server_manual/services/ogcapif.rst (hunk 1 above)
Comment (DelazJ) [reply docs/server_manual/services/ogcapif.rst:441]: I should revert until it's fixed in sources? Or do you think it's something that can be done quite quickly. if latter, would that be backported to LTR also?

Comment (elpaso) [reply docs/server_manual/services/ogcapif.rst:441]: better revert it

**Pull request merged**